h_transport_materials/property_database/eurofer_97/eurofer_97.py

```diff
@@ -0,0 +1,183 @@
+    delimiter=",",
+)
+
+aiello_permeability_data_H = aiello_permeability_data[2:, :2]
+
+aiello_permeability_H = Permeability(
+    data_T=1000 / aiello_permeability_data_H[:, 0] * u.K,
+    data_y=aiello_permeability_data_H[:, 1]
+    * u.mol
+    * u.m**-1
+    * u.Pa**-0.5
+    * u.s**-1,
+    source="aiello_hydrogen_2002",
+    isotope="H",
+)
+
+aiello_permeability_data_D = aiello_permeability_data[2:, 2:]
+
+aiello_permeability_D = Permeability(
+    data_T=1000 / aiello_permeability_data_D[:, 0] * u.K,
+    data_y=aiello_permeability_data_D[:, 1]
+    * u.mol
+    * u.m**-1
+    * u.Pa**-0.5
+    * u.s**-1,
+    source="aiello_hydrogen_2002",
+    isotope="D",
+)
+
+aiello_diffusivity_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/aiello_2002/permeability.csv",
+    delimiter=",",
+)
+
+aiello_diffusivity_data_H = aiello_diffusivity_data[2:, :2]
+
+aiello_diffusivity_H = Diffusivity(
+    data_T=1000 / aiello_diffusivity_data_H[:, 0] * u.K,
+    data_y=aiello_diffusivity_data_H[:, 1] * u.m**2 * u.s**-1,
+    source="aiello_hydrogen_2002",
+    isotope="H",
+)
+
+aiello_diffusivity_data_D = aiello_diffusivity_data[2:, 2:]
+
+aiello_diffusivity_D = Diffusivity(
+    data_T=1000 / aiello_diffusivity_data_D[:, 0] * u.K,
+    data_y=aiello_diffusivity_data_D[:, 1] * u.m**2 * u.s**-1,
+    source="aiello_hydrogen_2002",
+    isotope="D",
+)
+
+chen_permeability_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/chen_2021/permeability.csv",
+    delimiter=",",
+)
+
+chen_permeability = Permeability(
+    data_T=1000 / chen_permeability_data[:, 0] * u.K,
+    data_y=chen_permeability_data[:, 1] * u.mol * u.m**-1 * u.MPa**-0.5 * u.s**-1,
+    source="chen_deuterium_2021",
+    isotope="D",
+)
+
+chen_diffusivity_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/chen_2021/diffusivity.csv",
+    delimiter=",",
+)
+
+chen_diffusivity = Diffusivity(
+    data_T=1000 / chen_diffusivity_data[:, 0] * u.K,
+    data_y=chen_diffusivity_data[:, 1] * u.m**2 * u.s**-1,
+    source="chen_deuterium_2021",
+    isotope="D",
+)
+
+chen_solubility = Solubility(
+    S_0=4.0e2 * u.mol * u.m**-3 * u.MPa**-0.5,
+    E_S=29.2 * u.kJ * u.mol**-1,
+    range=(623 * u.K, 823 * u.K),
+    note="Given in paper chen_deuterium_2021 bbut no experimental data points",
+    source="chen_deuterium_2021",
+    isotope="D",
+)
+
+esteban_permeability_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/esteban_2007/permeability.csv",
+    delimiter=",",
+)
+
+esteban_permeability = Permeability(
+    data_T=1000 / esteban_permeability_data[:, 0] * u.K,
+    data_y=esteban_permeability_data[:, 1]
+    * u.mol
+    * u.m**-1
+    * u.Pa**-0.5
+    * u.s**-1,
+    source="esteban_hydrogen_2007",
+    isotope="H",
+)
+
+esteban_diffusivity = Diffusivity(
+    D_0=4.57e-07 * u.m**2 * u.s**-1,
+    E_D=22.3 * u.kJ * u.mol**-1,
+    range=(376 * u.K, 724 * u.K),
+    note="Data given in esteban_hydrogen_2007 as effective diffusivity",
```

Comment:
This is confusing imo: is this an effective diff?

Maybe "The authors also give an effective diffusivity, this is the lattice diffusivity measured at high temperature"